keyform/templates/keyform/contact_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'keyform/base.html' %}
+{% load i18n %}
+{% load static %}
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' href='{% static "keyform/css/contact_form.css" %}'>
+{% endblock %}
+
+{% block content %}
+    <h2 class='title'>{{ title }} {% trans 'Contact' %}</h2>
```

Comment:
Instead of `{{ title }} {% trans 'Contact' %}`, use:

```html
{% if form.instance.pk %}{% trans 'Edit Contact' %}{% else %}{% trans 'Create Contact' %}{% endif %}
```